.github/workflows/cloudflare-preview.yml

```diff
@@ -0,0 +1,36 @@
+on:
+  pull_request:
+    branches:
+      - develop
+name: Cloudflare-preview
+jobs:
+  build-deploy:
```

Comment:
Please choose a unique job name, as this name is already used in this repo. I would suggest `cloudflare-preview`.